adapters/syncswap/src/index.ts

```diff
@@ -0,0 +1,64 @@
+import { getPositionsForAddressByPoolAtBlock as getSyncSwapPositionsForAddressByPoolAtBlock} from "./sdk/positionSnapshots"
+
+import { promisify } from 'util';
+import stream from 'stream';
+import csv from 'csv-parser';
+import fs from 'fs';
+import { write } from 'fast-csv';
+
+
+interface CSVRow {
```

Comment:
Can you please change the output CSVRow to match the data schema outlined in the readme, and make the logic changes needed to get data in the desired schema.

I understand the confusion here due to the example. We have changed the schema for this project and need every protocol to adhere to it.

Thank you for your cooperation.

https://github.com/delta-hq/l2-lxp-liquidity-reward?tab=readme-ov-file#output-data-schema